todo_app/data/Trello_items.py

```diff
@@ -0,0 +1,75 @@
+import requests
+import os
+
+from todo_app.item import Item
+
+def add_item(title):
+    import ssl
+    context = ssl.create_default_context()
+    der_certs = context.get_ca_certs(binary_form=True)
+    pem_certs = [ssl.DER_cert_to_PEM_cert(der) for der in der_certs]
+    with open('wincacerts.pem', 'w') as outfile:
+        for pem in pem_certs:
+            outfile.write(pem + '\n')
+
+    reqUrl = "https://api.trello.com/1/cards"
+
+    query_params = {
+        "key" : os.getenv("TRELLO_API_KEY") ,
+        "token" : os.getenv("TRELLO_API_TOKEN") ,
+        "idList" : os.getenv("TRELLO_TODO_LIST_ID") ,
+        "name" : title
+    }
+
+    response = requests.post(reqUrl, params=query_params, verify='wincacerts.pem')
+
+    print(response.text)
```

Comment:
Very minor, but note that this prints out all the details related to the new card - in particular this means that if we were to release this application into production then currently we'll end up with potentially sensitive data in our logs (e.g. it's easy to imagine a user might have a todo like "speak to GP about <medical condition>" or "send email to friend at <address>"). Now obviously this doesn't matter much right now, when anyone will be able to access and view those to-dos anyway, but if we later introduce user accounts then there's a real risk that we can't differentiate between who has access to our logs, vs who has access to production data, and we have to treat all our logs as highly sensitive

To push this further, imagine if you discovered that all your Facebook messages were being printed out in their logs. I'd definitely be concerned about that security!

So really not a problem for this app, but nonetheless I'd try stick to the habit of not printing out potentially sensitive data, and removing this